src/plonk/circuit.rs

```diff
@@ -11,7 +11,7 @@ use crate::poly::Rotation;
 pub trait ColumnType: 'static + Sized {}
 
 /// A column with an index and type
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
```

Comment:
`Hash` is derived here so we can use columns with a `HashSet` or as keys in a `HashMap`.